docker/Dockerfile

```diff
@@ -17,3 +19,15 @@ RUN install2.r --error remotes &&  \
 
 COPY . /src
 RUN R CMD INSTALL --install-tests /src && rm -rf /src
+
+ENV ORDERLY_RUNNER_QUEUE_ID=orderly.runner
+ENV REDIS_CONTAINER_NAME=orderly.runner_redis
```

Comment:
I don't think that this should be a default envvar in the image

Reply:
removed